en/releases/release-1.1.12.md

```diff
@@ -0,0 +1,29 @@
+---
+title: TiDB Operator 1.1.12 Release Notes
+---
+
+# TiDB Operator 1.1.12 Release Notes
+
+Release date: April 15, 2021
+
+TiDB Operator version: 1.1.12
+
+## New Features
+
+- Support setting customized `ENV` for `Backup` and `Restore` jobs ([#3833](https://github.com/pingcap/tidb-operator/pull/3833), [@dragonly](https://github.com/dragonly))
```

Comment:
Can you align these notes with [1.2.0-beta.1](https://docs.pingcap.com/tidb-in-kubernetes/dev/release-1.2.0-beta.1)? Most of them are the same.

Reply:
done in https://github.com/pingcap/docs-tidb-operator/pull/1161/commits/5241972df511674286d8bc89a786b40159868b18